moveos/moveos-stdlib/moveos-stdlib/sources/move_module.move

```diff
@@ -55,15 +55,45 @@ module moveos_std::move_module {
         check_compatibililty_inner(new_module.byte_codes, old_module.byte_codes);
     }
 
-    /// Remap addresses in module binary where the length of
-    /// `old_addresses` must equal to that of `new_addresses`.
-    public fun remap_module_addresses(
+    /// Binding given module's address to the new address
+    public fun binding_module_address(
         modules: vector<MoveModule>,
-        old_addresses: vector<address>,
-        new_addresses: vector<address>,
+        old_address: address,
+        new_address: address,
+    ): vector<MoveModule> {
+        let bytes_vec = vector::empty<vector<u8>>();
+        let i = 0u64;
+        let len = vector::length(&modules);
+        while (i < len) {
+            vector::push_back(&mut bytes_vec, vector::pop_back(&mut modules).byte_codes);
+            i = i + 1;
+        };
+        let old_addresses = vector::singleton(old_address);
+        let new_addresses = vector::singleton(new_address);
+        
+        let rebinded_bytes = replace_address_identifiers(bytes_vec, old_addresses, new_addresses);
+        let rebinded_bytes = replace_addresses_constant(rebinded_bytes, old_addresses, new_addresses);
+        let rebinded_modules = vector::empty<MoveModule>();
+        i = 0u64;
+        let len = vector::length(&rebinded_bytes);
+        while (i < len) {
+            vector::push_back(&mut rebinded_modules, MoveModule {
+                byte_codes: vector::pop_back(&mut rebinded_bytes),
+            });
+            i = i + 1;
+        };
+        vector::destroy_empty(rebinded_bytes);
+        rebinded_modules
+    }
+
+    /// Replace given module's name to the new name
+    public fun replace_module_name(
```

Comment:
The name may be some other identifier, so should we rename this function to `replace_module_identiner` to avoid misuse?